### PR TITLE
add clickspark.js@1.16.0 w/ npm auto-update

### LINKED ISSUE #8459: [Request] Add ClickSpark.js
**Library name:** ClickSpark.js
https://github.com/ymc-thzi/clickspark.js
$ npm install clickspark.js
MIT License
http://www.ymc.ch/sandbox/clickspark/demo.html

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8459 
@pvnr0082t  please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ymc-thzi/clickspark.js
 * Official website (optional, not the repository): http://www.ymc.ch/sandbox/clickspark/demo.html
 * NPM package url (optional): https://www.npmjs.com/package/clickspark.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 253
   - Count of watchers: 18
   - Count of forks: 40
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 28
   - Downloads in the last month: 73

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8459, cc @ymc-thzi

### CODE REVIEW

File: ajax/libs/clickspark.js/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "dependencies": {
+    "jquery": "~2.1.4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ymc-thzi/clickspark.js.git"
+  },
+  "author": "Thomas Zinnbauer",
+  "license": "MIT",
+  "homepage": "http://www.ymc.ch/sandbox/clickspark/demo.html",
```
Comment (pvnr0082t) [inline ajax/libs/clickspark.js/package.json]: @kennynaoh I think it's for `demo` purpose. Please remove it.

Comment (kennynaoh) [inline ajax/libs/clickspark.js/package.json]: But I think it's ok.
cc @PeterDaveHello

Comment (PeterDaveHello) [inline ajax/libs/clickspark.js/package.json]: Looks okay.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**